### DISCUSSION TITLE
signInWithOAuth has thrown "Internal server error"

### DISCUSSION THREAD

**Discussion by kiuthy229** [supabase/supabase#22124] [Questions]
I tried to sign in with Supabase Google Authn but turned out it redirects to an empty page with error
```json
{"code":500,"msg":"Internal server error","error_id":"86732ced27223dab-SIN"}
```
<img width="896" alt="Screenshot 2024-03-20 at 12 09 30" src="https://github.com/supabase/supabase/assets/68453603/1dc1b582-cde0-49c1-a186-7ecf0fe4c9e5">

My code:
<img width="506" alt="Screenshot 2024-03-20 at 12 09 58" src="https://github.com/supabase/supabase/assets/68453603/060ebd2b-fd6b-47d9-9005-ffa9a906a164">

**Comment by GaryAustin1** [comment]
Usually a 500 error on signUp/in involves you having a trigger function on auth.users and that function getting an error.  Please check the postgres and/or auth logs in the dashboard to find the details on the 500 error if that is the case.
